### PR TITLE
94703-adding-complete-color-palette

### PR DESCRIPTION
**Pull request by boskya**
94703
### Review
 - [x] @jtrinklein

### After Merge

- [ ] [site](https://daptiv.github.io/style-guide) is updated

### CODE REVIEW

File: docs/docs.scss [hunk 1]
```diff
@@ -1,22 +1,104 @@
...
-    border-radius: 0.25rem;
-    border-color: black;
+    width: $swatch-width-height;
+    height: $swatch-width-height + $swatch-label-height + 2px;
+    vertical-align: top;
+    margin: 4px;
+
+    label{
+        background-color:white;
+        width:$swatch-width-height;
```
Comment (jtrinklein) [inline docs/docs.scss]: :+1:

File: docs/docs.scss [hunk 2]
```diff
@@ -1,22 +1,104 @@
...
+        display:block;
+        height: $swatch-label-height;
+        font-size: 1.2em;
+        margin: 4px;
```
Comment (jtrinklein) [inline docs/docs.scss]: can we make a variable for this? `$standard-whitespacing` or something?

File: docs/docs.scss [hunk 3]
```diff
@@ -1,22 +1,104 @@
...
+.swatch-dark-gray {
+    background-color: $color-dark-gray;
+}
+
+.swatch-medium-dark-gray {
+    background-color: $color-medium-dark-gray;
+}
+
+.swatch-medium-gray {
+    background-color: $color-medium-gray;
```
Comment (jtrinklein) [inline docs/docs.scss]: no hound, it should be 4

**Comment by jtrinklein** [comment]
just make hound happy :hurtrealbad:

**Comment by jtrinklein** [comment]
then I'm :+1:

**Comment by boskya** [comment]
Hound is mostly happy now.

**Pull request merged**